target/consensource-api
# Remove 'batch_id' and 'transaction_id' from user model

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

Remove the batch_id and transaction_id fields from the User table/model schemas. We are not storing User objects on-chain so these fields should not exist.

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- Update `Cargo.toml` database dependency branch to `perichsen-user-model-update`
- `cd test`
- `docker-compose up` and check that all tests passed in tarpaulin
